libs/dh/market-participant/actors/feature-actors/src/lib/drawer/dh-actor-drawer.component.ts

```diff
@@ -92,7 +117,25 @@ export class DhActorDrawerComponent {
     query: GetActorByIdDocument,
   });
 
+  private getActorAuditLogByIdQuery$ = this.apollo.watchQuery({
+    errorPolicy: 'all',
+    returnPartialData: true,
+    useInitialLoading: true,
+    notifyOnNetworkStatusChange: true,
+    query: GetAuditLogByActorIdDocument,
+  });
+
   actor: DhActorExtended | undefined = undefined;
+  isLoadingAuditLog = false;
+  auditLogFailedToLoad = false;
+
+  auditLog: WattTableDataSource<ActorAuditLogEntry> = new WattTableDataSource<ActorAuditLogEntry>(
+    []
+  );
```

Comment:
nitpick: the type is automatically inferred by TS.


```suggestion
  auditLog = new WattTableDataSource<ActorAuditLogEntry>([]);
```